cmd/markdown/atmos_toolchain_aliases.md

```
@@ -0,0 +1,4 @@
+- Get the aliases configured
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)